apps/condo/domains/common/components/TicketCard/TicketCardList.tsx

```diff
@@ -0,0 +1,169 @@
+
+const TicketCard: React.FC<ITicketCardProps> = ({
+    address,
+    tickets,
+    contactName,
+}) => {
+    const intl = useIntl()
+    const AddressLabel = intl.formatMessage({ id: 'field.Address' })
+    const DeletedMessage = intl.formatMessage({ id: 'Deleted' })
+    const TicketsByContactMessage = intl.formatMessage({ id: 'TicketsByContact' })
+    const NoTicketsOnAddressMessage = intl.formatMessage({ id: 'Contact.NoTicketOnAddress' })
+
+    const addressPrefix = address ? address : DeletedMessage
+    const hasMoreTickets = tickets.length >= TICKETS_ON_CARD ? tickets.length - TICKETS_ON_CARD : 0
+
+    const filters = { clientName: contactName, address }
+    const query = qs.stringify(
+        {
+            filters: JSON.stringify(pickBy(filters)),
+        },
+        { arrayFormat: 'comma', skipNulls: true, addQueryPrefix: true },
+    )
+
+    return (
+        <Container >
+            <AddressPartContainer>
+                <Space size={8} direction={'vertical'}>
+                    <Typography.Text type='secondary'>
+                        {AddressLabel}
+                    </Typography.Text>
+                    <Typography.Title level={4}>
+                        {addressPrefix}
+                    </Typography.Title>
+                </Space>
+            </AddressPartContainer>
+            <TicketsPartContainer>
+                <Row gutter={TICKET_CARD_GUTTER}>
+                    <Col span={24}>
+                        <>
+                            {tickets.length > 0
+                                ?
+                                <Typography.Title level={5} >
+                                    {TicketsByContactMessage}
+                                </Typography.Title>
+                                :
+                                <Typography.Text>
+                                    {NoTicketsOnAddressMessage}
+                                </Typography.Text>
+                            }
+                            {
+                                tickets.slice(0, TICKETS_ON_CARD).map((ticket) => {
+                                    return <TicketOverview
+                                        key={ticket.id}
+                                        id={ticket.id}
+                                        details={ticket.details}
+                                        createdAt={ticket.createdAt}
+                                        number={ticket.number}
+                                        status={ticket.status.name}/>
+                                })
+                            }
+                            {hasMoreTickets > 0 && (
+                                <Col span={24}>
+                                    <Link href={`/ticket/${query}`}>
+                                        <Typography.Link style={TICKET_CARD_HAS_MORE_LINK_STYLE}>
+                                            {intl.formatMessage({ id: 'MoreTicketsLeft' }, {
```

Comment:
looks like can be memoized

Reply:
@akarjakin it can, but should it? There is no special logic to update this component. So there is only 1 render and it seems to me that cast useMemo hook on each piece of code, that might be rerendered is overhead here